compiler/rustc_attr/messages.ftl

```diff
@@ -25,7 +25,7 @@ attr_incorrect_meta_item =
     incorrect meta item
 
 attr_incorrect_repr_format_align_one_arg =
-    incorrect `repr(align)` attribute format: `align` takes exactly one argument in parentheses
+    incorrect `repr(align)` attribute format: `align` takes exactly one argument in parentheses, namely the number of desired alignment bytes
```

Comment:
The error message when there is N=1 argument (but it's the wrong kind) should be broken out into a separate diagnostic, I think. I wonder if we could even special-case the `i*`/`u*` types and suggest how to fix it (chop off the `i` or `u` and divide it by 8?)

Reply:
Agreed that if the ident is the name of a primitive, suggesting the size of that primitive would be nice.

Reply:
I like both of those ideas, although I have no idea how to implement either of them. 😅